test/test_parse.py

```diff
@@ -71,20 +71,107 @@ def test_articles(self):
                                    remove_articles=False),
                          "this is an extra test")
 
-    def test_extract_number(self):
+    def test_extract_number_ambiguous(self):
+        # test explicit ordinals
+        self.assertEqual(extract_number("this is the 1st",
+                                        ordinals=True), 1)
+        self.assertEqual(extract_number("this is the 2nd",
+                                        ordinals=False), 2)
+        self.assertEqual(extract_number("this is the 3rd",
+                                        ordinals=None), 3)
+        self.assertEqual(extract_number("this is the 4th",
+                                        ordinals=None), 4)
+
+        # test non ambiguous ordinals
         self.assertEqual(extract_number("this is the first test",
                                         ordinals=True), 1)
-        self.assertEqual(extract_number("this is 2 test"), 2)
+        self.assertEqual(extract_number("this is the first test",
+                                        ordinals=False), False)
+        self.assertEqual(extract_number("this is the first test",
+                                        ordinals=None), False)
+
+        # test ambiguous ordinal/time unit
         self.assertEqual(extract_number("this is second test",
                                         ordinals=True), 2)
-        self.assertEqual(extract_number("this is the third test"), 1.0 / 3.0)
+        self.assertEqual(extract_number("this is second test",
+                                        ordinals=False), False)
+        self.assertEqual(extract_number("remind me in a second",
+                                        ordinals=True), 2)
+        self.assertEqual(extract_number("remind me in a second",
+                                        ordinals=False), False)
+        self.assertEqual(extract_number("remind me in a second",
+                                        ordinals=None), False)
+
+        # test ambiguous ordinal/fractional
         self.assertEqual(extract_number("this is the third test",
                                         ordinals=True), 3.0)
+        self.assertEqual(extract_number("this is the third test"),
+                         1.0 / 3.0)
+        self.assertEqual(extract_number("this is the third test",
+                                        ordinals=None), False)
+
+        self.assertEqual(extract_number("one third of a cup"), 1.0 / 3.0)
+        self.assertEqual(extract_number("one third of a cup",
+                                        ordinals=True), 3)
+        self.assertEqual(extract_number("one third of a cup",
+                                        ordinals=None), 1)
+
+        # test plurals
+        # NOTE plurals are never considered ordinals, but also not
```

Comment:
@ChanceNCounter @krisgesling does this note make sense?

Reply:
Since you have to extract ordinals explicitly, and there's no such ordinal as The Three Fifth, it's probably sensible to avoid returning fractions in all cases.

On the other hand, it might make sense to call them ordinals, on the assumption that `extract_number("the thirds", ordinals=True)` will always be an input failure somewhere upstairs, like STT mishearing "the third."

Reply:
i was unsure of this one, for consistency i thought it made sense that ordinals=True would not parse fractions at all, the same way ordinals=False would only parse fractions.  But since we support explict ordinals (1st, 2nd, 3rd, 4th...) i'm not sure....

i guess its a matter of how explicit is this? what are the possible sources of ambiguity?

"the thirds" could very well mean 3 instead of 1/3 , so i don't think this should be considered an explicit fraction

Reply:
i wonder if when ordinals=True we should consider the plural form an integer?